android/samples/test-app/src/main/res/values/server_context.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-	<string name="liferay_server">http://screens-demo.liferay.org.es</string>
+	<string name="liferay_server">http://screens.liferay.org.es</string>
```

Comment:
Exactly the same as the previous comment. I haven't deleted the commit, will pick the commit when github does the merge